snippets/pyaloha/event_factory.py

```diff
@@ -24,10 +25,14 @@ def make_event(self, key, *args, **kwargs):
                 )
             )
 
+    @staticmethod
+    def get_duplicates(list_to_check):
+        return [m[0] for m in filter(lambda elem: elem[1] > 1, collections.Counter(list_to_check).items())]
```

Comment:
return [
    value
    for value, count in Counter(list_to_check).items()
    if count > 1
]